modules/nodes-cluster-enabling-features-console.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-cluster-enabling-features.adoc
+
+[id="nodes-cluster-enabling-features-console_{context}"]
+= Enabling feature sets using the web console
+
+You can use the {product-title} web console to enable feature sets for all the nodes in a cluster by editing the `FeatureGate` custom resource (CR).
+
+.Procedure
+
+To enable feature sets:
+
+. In the {product-title} web console, switch to the *Administration* -> *Custom Resource Definitions* page.
```

Comment:
What do you think about simplifying this to:

> In the **Administrator** perspective, navigate to the **Custom Resource Definitions** page.

Reply:
Which version did we introduce the Administrator perspective (so that's what it's called!). This change need to be CPd back to 4.5. Would rather not do multiple versions. (Although, this should be a global change, come to think about it.)